tempesta_fw/tls.c

```diff
@@ -530,8 +539,20 @@ tfw_cfgop_cleanup_ssl_certificate_key(TfwCfgSpec *cs)
 static int
 tfw_tls_cfgend(void)
 {
-	if ((tfw_tls.crt.version && !tfw_tls.key.pk_ctx) ||
-	    (!tfw_tls.crt.version && tfw_tls.key.pk_ctx)) {
+	if (!(tfw_tls_cgf & TFW_TLS_CFG_F_REQUIRED)) {
+		if (tfw_tls_cgf & TFW_TLS_CFG_F_REQUIRED)
```

Comment:
Condition is always false: `if (!a) { if (a) { .. } }`. Bad copy-paste?
`if (tfw_tls_cgf & TFW_TLS_CFG_F_REQUIRED)` -> `if (tfw_tls_cgf)`